pandas/hashtable.pyx

```diff
@@ -399,137 +324,166 @@ cdef class Int64HashTable: #(HashTable):
         else:
             raise KeyError(key)
 
-    def map(self, ndarray[int64_t] keys, ndarray[int64_t] values):
+    @cython.boundscheck(False)
```

Comment:
yes when you release the gil it gives a warning if you don't do this (as you are de-facto turning it off in c-land anyhow, I think it wants to make it explict)